framework/test/driver/BundleTest.cpp

```diff
@@ -879,5 +1083,18 @@ int BundleTest(int /*argc*/, char* /*argv*/[])
   TestBundleGetPropertyKeys();
   TestBundleGetProperty();
 
+#if defined (US_BUILD_SHARED_LIBS)
```

Comment:
Why are these tests invalid for static builds?

Reply:
I think only ```TestBundleManifestFailures``` should be in this ```#ifdef``` block since all statically linked bundles would be installed when the ```Framework``` starts and the bundle designed to have an invalid manifest would fail all the tests. I'll double check and update.

Reply:
```TestBundleStopFail``` and ```TestBundleStartFail``` can't be built for static builds only because of the statically linked bundle tests in ```TestBundleStates()```.